docs/source/release_notes/v0.2.0.md

```diff
@@ -0,0 +1,33 @@
+# Version 0.2.0 (November 23, 2021)
+
+This is a new and enhanced version of MONAI Deploy App SDK, just in time for Thanksgiving and RSNA 2021!🎉
+
+Please visit [GETTING STARTED](/getting_started/index) guide and follow tutorials.
+
+You can learn more about SDK usage through [DEVELOPING WITH SDK](/developing_with_sdk/index).
+
+Please let us know how you like it and what could be improved by [submitting an issue](https://github.com/Project-MONAI/monai-deploy-app-sdk/issues/new/choose) or [asking questions](https://github.com/Project-MONAI/monai-deploy-app-sdk/discussions)😀
+
+## What's new in this version 0.2.0
+### Series Selection Operator
+This is to support the use case where whole DICOM studies are used as input to an AI inference application even though only specific series are applicable.
+
+The selection rules are defined in JSON, allowing multiple selections, each with a set of matching conditions. The rules processing engine is implemented in the `DICOMSeriesSelectorOperator`, which itself is regarded as a base class with default implementation. More advanced rules and processing engine can be implemented in derived classes.
```

Comment:
```suggestion
The selection rules are defined in JSON, allowing multiple selections, each with a set of matching conditions. The rules processing engine is implemented in the `DICOMSeriesSelectorOperator`, which itself is regarded as a base class with the default implementation. More advanced rules and processing engines can be implemented in derived classes.
```

Reply:
It was meant to say "rules processing engine"

Reply:
One second thought, rules can be redefined too, if the base class is not used.